core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponse.java

```diff
@@ -67,5 +67,6 @@ public boolean isShardsAcked() {
 
     public void addCustomFields(XContentBuilder builder) throws IOException {
         builder.field("shards_acknowledged", isShardsAcked());
+        
```

Comment:
I do this sometimes when I'm making a PR. I tend to revert all the changes I make to a file when I realize that I've put it back how I found it. This kind of thing is so common we have a term for it: "leftover".